ci/cpu/build.sh

```diff
@@ -15,6 +15,11 @@ export HOME=$WORKSPACE
 # Switch to project root; also root of repo checkout
 cd $WORKSPACE
 
+# If nightly build, append current YYMMDD to version
+if [[ "$BUILD_MODE" = "branch" && "$SOURCE_BRANCH" = branch-* ]] ; then
+  export VERSION_SUFFIX=`date +%y%m%d`
+fi
```

Comment:
Analogous to [this code in cuDF]( https://github.com/rapidsai/cudf/blob/b926f51dde17db1c591fa5d25c41a9f6b989233b/ci/cpu/build.sh#L32-L35 )

`VERSION_SUFFIX` is already in the recipe version (see below). It just wasn't being set before

https://github.com/rapidsai/ucx-py/blob/0caf1f2abf5475777dd3fc962e81697eec9f180a/conda/recipes/ucx-py/meta.yaml#L1